### PR TITLE
fix(content): misc fixes and mes adjustments

### PR DESCRIPTION
**Pull request by tannerdino**
- Smithing on first smith didnt have an extra delay after smithing the item. To match this video: https://youtu.be/vQixsBb7wPQ?t=95 (level up occurs 3 ticks after item is received) 
- Fishing spots in the fisher realm were incorrectly coded as net/harpoon
- Remove area checks in the fishing trawler teleport queues to match: https://youtu.be/RZwuEjlkVhY?t=148
- Check for lowmemory was missing, im not sure to keep the "memery" typo since this is an rs2 beta screenshot
image
- "You pay the 30 coins and board the ship." -> "You pay the 30 coins..." "and board the ship."
image
- Double space after "beer."
image
image
- Blue period at the end 
image
- Ned not restricted to outdoors
image

image
- Kbd poison and icy attack flipped, (Poison attack should be from its left side)
image
- Kbd fire blast spot anim on impact
image

### CODE REVIEW

**Comment by toomykins** [comment]
#890 also alters ned's indoor behavior. might lead to merge issues

File: data/src/scripts/tutorial/scripts/tut_chatbox_steps.rs2 [hunk 1]
```diff
@@ -94,7 +94,11 @@ if (lowmemory = true) {
 
 [proc,tutorial_step_music_player]
 hint_coord(^hint_east, 0_48_48_0_18, 150);
-~tutorialstep("The Music Player.", "From this interface you can control the music that is played.|As you explore the world more of the tunes will become unlocked.|Once you've examined this menu use the next door|to continue.");
+if (lowmemory = true) { // https://web.archive.org/web/20040408025047/http://www.imageshack.us:80/img2/8038/runescapetypo1.png
+    ~tutorialstep("The Music Player.", "As you are playing in low memory mode you will not be able to|listen to the music of RuneScape, but this is where you would|find the jukebox controls. Pass through the door to continue. If you need a recap on anything covered here tal to Lev.");
```
Comment (Indio3) [inline data/src/scripts/tutorial/scripts/tut_chatbox_steps.rs2:98]: should be "talk to Lev."

Comment (tannerdino) [reply data/src/scripts/tutorial/scripts/tut_chatbox_steps.rs2:98]: Ty, do you think it should be "memery" or "memory" btw

Comment (Indio3) [reply data/src/scripts/tutorial/scripts/tut_chatbox_steps.rs2]: "memery" is probably correct, I don't think that would have been an edit made in 2004

Comment (Indio3) [reply data/src/scripts/tutorial/scripts/tut_chatbox_steps.rs2]: imo I don't think they would of left an obvious typo in the tutorial for that long, so might've been fixed by the point they added progress bar?

**Pull request merged**